index.html

```diff
@@ -0,0 +1,227 @@
+                        <li id="lang1">HTML</li>
+                        <li id="lang2">Bootstrap</li>
+                        <li id="lang3">Ruby</li>
+                    </ul>
+                </article>
+                <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+                    Project</button>
+        </section>
+        <section class="project" id="prcjt6">
+            <h3 class="project_descp1">Data Dashboard Healthcare</h3>
+            <p class="project_detail">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                been the industry's standard</p>
+                <article class="lang_container_2">
+                    <ul style="list-style-type:none;" class="list">
+                        <li id="lang1">HTML</li>
+                        <li id="lang2">Bootstrap</li>
+                        <li id="lang3">Ruby</li>
+                    </ul>
+                </article>
+                <button type="submit" class="checkout-button" id="projects-btn2" name="checkout-button" enable="disabled">See
+                    Project</button>
+        </section>
+    </section>
+    <section class="about_me">
+        <h2 id="about_text">About Me</h2>
+        <p id="intro_aboutme">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+        <button type="submit" class="checkout-button" id="resume-btn" name="checkout-button" enable="disabled">Get My
+            Resume</button>
+        <hr id="divider">
+        <section id="lang_sec">
+            <img id="rect" src="/attributes/Rectangle 55.png" />
+            <p class="lang_use">Language</p>
+        </section>
+
+        <section class="artc_lang">
+            <ul style="list-style-type:none;">
+                <li class="lang_btn" id="langbtn1">Javascript</li>
+                <li class="lang_btn" id="langbtn2">Ruby</li>
+                <li class="lang_btn" id="langbtn3">HTML</li>
+                <li class="lang_btn" id="langbtn4">CSS</li>
+            </ul>
+        </section>
+
+        <section id="fram_sec">
+            <img id="sqr" src="/attributes/Rectangle 55.png" />
+            <p class="lang_use">Frameworks</p>
+        </section>
+        <section class="artc_lang" id="framework_sec">
+            <ul style="list-style-type:none;">
+                <li class="lang_btn" id="frambtn1">Bootstrap</li>
+                <li class="lang_btn" id="frambtn2">Ruby on Rails</li>
+                <li class="lang_btn" id="frambtn3">RSPec</li>
+                <li class="lang_btn" id="frambtn4">CapyBara</li>
+                <li class="lang_btn" id="frambtn5">Selenium</li>
+            </ul>
+        </section>
```

Comment:
[OPTIONAL] Per the layout of the figma design, this project is suppose to have the `top navigation`, `header/headline`, `projects section`, `about myself section` and `the footer(contact me)`. Semantically and for best practices, you can use a nav tag for the top navigation, header tag for the header/headline, section tags for both the projects and about myself sections, and  footer tag for the contact me. :1234: 
There is no need to sub divide the sections into subsections using the section tags as has been done here. Kindly use the article tag or div tag for all other parts which are not sections. This will help you to achieve the best optimum solution semantically as well as help your project to be more proffessional. :+1: